pkgs/servers/web-apps/engelsystem/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  stdenv
+, fetchzip
+, php
+}:
+
+  stdenv.mkDerivation rec {
+    pname = "engelsystem";
+    version = "3.1.0";
+
+     src = fetchzip {
+      url = "https://github.com/engelsystem/engelsystem/releases/download/v3.1.0/engelsystem-v3.1.0.zip";
```

Comment:
Use fetchFromGitHub if possible

Reply:
it uses the artifact, uploaded by an engelsystem maintainer